doc/manuals/simulation-configuration.md

```diff
@@ -1,9 +1,95 @@
-# Simulation configuration file
+# Simulation configuration
+
+## Environment variables file
+
+The simulation configuration file may use an .env file to set certain configuration on simulation.json file, this is used to avoid set sensitive information, and be executed in whatever environment, having different configurations and not been needed to change anything on code.
+
+Here's an example of `.env` file with the commonly environment variables used:
+
+```json
+AUTH_PASS="pass"
+AUTH_USER="user"
+AUTH_PORT=5001
+AUTH_HOST="localhost"
+
+DOMAIN_SERVICE="service"
+DOMAIN_SUBSERVICE="/subservice"
+
+CB_HOST="localhost"
+CB_PORT=1026
+```
+
+To use a environment variable on the configuration file, it's needed to set the prefix `process.env.` and right after the variable name `{VARIABLE}` having as result `process.env.{VARIABLE}`.
+
+An example simulation configuration file using environment variables is shown next to give you a glimpse of its shape:
+
+```json
+{
+  "domain": {
+    "service": "process.env.DOMAIN_SERVICE",
+    "subservice": "process.env.DOMAIN_SUBSERVICE"
+  },
+  "contextBroker": {
+    "protocol": "http",
+    "host": "process.env.CB_HOST",
+    "port": "process.env.CB_PORT",
```

Comment:
It would be more "natural" something like this:

```
    "host": "${CB_HOST}",
    "port": "${CB_PORT}",
```

Reply:
Fixed